src/data_types/test_dictionaries.py

```diff
@@ -77,3 +77,6 @@ def test_dictionary():
     assert dictionary_for_string_keys['sape'] == 4139
     assert dictionary_for_string_keys['guido'] == 4127
     assert dictionary_for_string_keys['jack'] == 4098
+
+
+test_dictionary()
```

Comment:
Missing a new line here and on other files too. Syntactically the python files or any other files it's reasonable to add an empty new line at the end of the files.

Reply:
This still has not empty new line at the end of file.